operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -5,14 +5,19 @@ is only relevant for booting compute nodes and can be ignored when working with
 
 This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image.
 
-1. (`ncn-mw#`) Locate the NCN image to be modified.
+1. Identify the NCN image to be modified.
 
-    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by Kubernetes NCNs. However, the steps are the same for any Management NCN SquashFS image.
+    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by Kubernetes NCNs.
```

Comment:
Maybe provide a brief use case for why an admin might want to do this.  Keywords that someone would search for that you would want to lead them here.

Reply:
I do agree with you, but unless you have a good suggested use case handy, I'll probably defer this to a future ticket (which I will open before this PR merges).

Reply:
This will be addressed with [CASMINST-5572](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5572)